distribution/src/main/release/samples/jax_rs/spring_boot/README

```diff
@@ -8,7 +8,7 @@ The sample uses Maven. It can be built and run from the command line:
 ----
 $ mvn -Pserver
 or
-$ java -jar target/spring-boot-sample-rs-cxf-3.2.0-SNAPSHOT.jar
+$ java -jar target/spring-boot-sample-rs-cxf-*.jar
```

Comment:
If I am not mistaken, this command may not work, you have to provide the exact runnable: `java -jar `ls target/*.jar`` (but than it won't work for Windows users)